app/services/mentions/create_all.rb

```diff
@@ -36,6 +36,11 @@ def delete_removed_mentions(usernames)
     def create_mention(user)
       mention = Mention.create(user_id: user.id, mentionable_id: @notifiable.id, mentionable_type: @notifiable.class.name)
       # mentionable_type = model that created the mention, user = user to be mentioned
+
+      # if parent comment auther is user than dont send Notification
+      # 1. get parrent comment
+      # 2. get auther of parrent comment
+      # 3. check in mentioned user is auther or not
```

Comment:
@blunket Mentions have attributes like  
```
#<Mention id: 25, created_at: "2019-10-10 08:57:47", mentionable_id: 22, mentionable_type: "Comment", updated_at: "2019-10-10 08:57:47", user_id: 28>
```
 but to solve this it requires to get comment in which mention is created. 
could you suggest me how to get access to comment from mention?

Reply:
Sorry for the late response, you can see that the mentionable_type is a "comment'. With that, you can use `Comment.find(22)` in rail console to get the comment it is referring to.